### ISSUE TITLE
DRB area features should be deactivated when not available for use

### ISSUE THREAD

**Issue by mmcfarland** [WikiWatershed/model-my-watershed#764]
Both the Stream slider, the Sensor Overlays and the Delineate Watershed features are/will be designed to work only in the Delaware River Basin area.  We need a method to disable these tools when the viewport is not in an appropriate zoom-level & bbox.

The boundary shape is located locally at `//fileshare/projects/Stroud_ModelMyWatershed/data/drb_bnd_polygon.zip`.

My initial thoughts are that the app should maintain a simplified version of that shape, or maybe it's bbox, and enable those tools when it intersects with the viewport bbox at a reasonably high zoom level that makes sense to view/use the tools.

**Comment by mmcfarland** [comment]
Blocked on #763

**Comment by mmcfarland** [comment]
Hide stream and Delineate on prd site.

**Comment by mmcfarland** [comment]
Focus on just delineate watershed at the moment, but the mechanism should support additional future features.  Let's talk about this before implementation.

**Comment by mmcfarland** [comment]
The features mentioned in this issue have evolved.  I'm closing this in favor or new issues associated with their individual features.
